azure-spring-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/utils/Utils.java

```diff
@@ -100,34 +107,27 @@ public static int convertSizeStringToNumber(String memory) throws MojoExecutionE
         }
     }
 
-    public static List<File> getArtifacts(List<Resource> resources) {
-        final List<File> result = new ArrayList<>();
-        final DirectoryScanner directoryScanner = new DirectoryScanner();
-        for (final Resource resource : resources) {
-            if (resource.getIncludes() != null && resource.getIncludes().size() > 0) {
-                directoryScanner.setBasedir(resource.getDirectory());
-                directoryScanner.setIncludes(resource.getIncludes().toArray(new String[0]));
-                final String[] exclude = resource.getExcludes() == null ? new String[0] :
-                        resource.getExcludes().toArray(new String[0]);
-                directoryScanner.setExcludes(exclude);
-                directoryScanner.scan();
-                final List<File> resourceFiles = Arrays.stream(directoryScanner.getIncludedFiles())
-                        .map(path -> new File(resource.getDirectory(), path))
-                        .collect(Collectors.toList());
-                result.addAll(resourceFiles);
-            }
-        }
-        return result;
-    }
-
     public static String generateTimestamp() {
         return new SimpleDateFormat(DATE_FORMAT).format(new Date());
     }
 
-    public static File getArtifactFromConfiguration(SpringConfiguration springConfiguration) {
+    public static File getArtifactFromTargetFolder(MavenProject project) throws MojoExecutionException {
+        try {
+            final String targetFolder = project.getBuild().getDirectory();
+            final List<File> files = Files.walk(Paths.get(targetFolder))
```

Comment:
https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#listFiles(java.io.File,%20java.lang.String[],%20boolean)

Reply:
Chang to use FileUtils